apps/reviews/templates/reviews/edit_review.html

```diff
@@ -1,19 +1,19 @@
 {% from "includes/forms.html" import pretty_field, required_note %}
 <div class="hidden">
-  <form method="post" id="review-edit-form" action="#"
+  <form method="post" id="{{ form.form_id() }}-form" action="#"
         class="review article review-form prettyform">
     {{ csrf() }}
     <fieldset>
       <ul>
-        {{ pretty_field(form.title, _('Title')) }}
-        {{ pretty_field(form.rating, _('Rating')) }}
-        {{ pretty_field(form.body, _('Review')) }}
+        {% for field in form %}
+        {{ pretty_field(field, _(field.label)) }}
```

Comment:
I would rather have the label marked as localizable in the form itself (see above comment)